app/(app)/home/main.tsx

```diff
@@ -71,7 +70,11 @@ const Main = () => {
 
       {user && (
         <>
-          <Space size={16} />
+          <Space size={10} />
+
+          <VideoPlayer />
```

Comment:
The video is no longer playing on iOS, can you have a look why. It is playing OK on Android. If it is a bug we can open a separate ticket for it since I think it is not related to this PR.

<img width="405" alt="image" src="https://github.com/droidconKE/droidconKE2023ReactNative/assets/5578688/a37bed34-2e6c-4791-b17c-82aa6dce2629">

Reply:
for me its playing on both emulator and ios device. I'll create a ticket if need be

Reply:
No need. I cleared the cache and now the video plays.